tree/ntuple/CMakeLists.txt

```diff
@@ -60,13 +61,22 @@ SOURCES
   v7/src/RPagePool.cxx
   v7/src/RPageStorage.cxx
   v7/src/RPageStorageFile.cxx
+  v7/src/RDaos.cxx
+  v7/src/RPageStorageDaos.cxx
 LINKDEF
   LinkDef.h
 DEPENDENCIES
   RIO
   ROOTVecOps
 )
 
+# TODO(jalopezg): make daos support optional
```

Comment:
Hey @jalopezg-r00t  I recomend you here to use `find_package(DAOS)` https://github.com/HDFGroup/vol-daos/blob/master/src/CMakeLists.txt#L18, but please add it `SearchInstalledSoftware.cmake` (as an example you can use `uring` section -> https://github.com/root-project/root/blob/master/cmake/modules/SearchInstalledSoftware.cmake#L1173-L1190). Then you will need to add https://github.com/HDFGroup/vol-daos/blob/master/CMake/FindDAOS.cmake here: https://github.com/root-project/root/tree/master/cmake/modules/. Then you will not need to hardcode anything :+1:  even while testing backend on Openlab nodes.

Reply:
Thanks, @oshadura!  I took that shortcut as we needed to present some results for the Intel DAOS User Group meeting last week.  Thanks for the information, it will save me some time!  :-)